CHANGELOG.markdown

```diff
@@ -1,3 +1,8 @@
+5.0.7 [2020.xx.yy]
+------------------
+* Move `FunctorWithIndex (TracedT m w)` instance from `lens`.
+  The new `indexed-traversable` can be disabled using the flag of the same name.
```

Comment:
This sentence reads a bit strangely to me. Perhaps rephrase it to this?

```suggestion
  This instance depends on the `indexed-traversable` package. This can be disabled using the flag of the same name.
```